types/office-js-preview/index.d.ts

```diff
@@ -582,8 +582,6 @@ declare namespace Office {
         /**
          * Provides access to the Outlook Add-in object model for Microsoft Outlook and Microsoft Outlook on the web.
          *
-         * [Api set: Mailbox 1.0]
```

Comment:
should we be replacing these with "[Api set: Mailbox 1.1]"?

Reply:
As we discussed offline, that would not be accurate for Mailbox 1.1 and I verified that all the reference docs, including 1.1, keep these APIs.